igvc/launch/sensors.launch

```diff
@@ -11,7 +11,15 @@
     <node name="light_controller" pkg="igvc" type="light_controller" output="screen"/>
 
     <!-- MOTOR CONTROLLER -->
-    <node name="motor_controller" pkg="igvc" type="motor_controller" />
+    <node name="motor_controller" pkg="igvc" type="motor_controller" output="screen">
+          <param name="p_l" type="double" value="12.0"/>
+          <param name="p_r" type="double" value="12.0"/>
+          <param name="d_l" type="double" value="0.5"/>
+          <param name="d_r" type="double" value="0.5"/>
+          <param name="i_r" type="double" value="0.0"/>
```

Comment:
When we were testing the motors last Friday afternoon, I noticed that the motor speed overshot the target slightly for full forward power and didn't quite reach the target for full backward power.  Setting the integral constant to something small (I would guess on the order of .001) would help fix these small errors.

Reply:
I am rather sure that I is not currently implemented properly in firmware so that is why I is 0.